circle.yml

```diff
@@ -63,14 +60,6 @@ jobs:
     steps:
       - attach_workspace:
           at: .
```

Comment:
I don't think this will work since it within another workflow. We need to add a ```checkout``` step. See [this](https://circleci.com/docs/2.0/configuration-reference/#checkout).

Reply:
Oh I didn't see that. I opened a PR for the checkout fix.